modules/dnn/test/test_layers.cpp

```diff
@@ -1949,6 +1949,9 @@ struct Layer_Test_Eltwise_bcast : testing::TestWithParam<tuple<string, int, tupl
         int backend = get<0>(backend_target);
         int target = get<1>(backend_target);
 
+        if (backend == DNN_BACKEND_CUDA && dim > 4)
+            applyTestTag(CV_TEST_TAG_LONG);
```

Comment:
BTW, Looks like no tests are skipped on CI.

Reply:
It takes less than 20s, so I use LONG tag

Reply:
Longer than 5s but less than 20s, so it should use LONG tag

![image](https://github.com/opencv/opencv/assets/49567307/b44a360a-0c5e-4b36-8e01-f0dd7fd8b835)

Reply:
Ok, lets run all CUDA tests.